src/auth/tests/credentials.rs

```diff
@@ -125,6 +128,15 @@ mod test {
         assert!(fmt.contains("ServiceAccountCredential"));
     }
 
+    #[tokio::test]
+    async fn create_api_key_credential_success() {
+        let creds = create_api_key_credential("test-api-key", ApiKeyOptions::default())
+            .await
+            .unwrap();
+        let fmt = format!("{:?}", creds);
+        assert!(fmt.contains("ApiKeyCredential"));
```

Comment:
maybe:

```suggestion
        assert!(fmt.contains("ApiKeyCredential"), "{fmt:?}");
```

but test that code before you rely on it.